### DISCUSSION TITLE
Deploy only Triggers - with no other dependencies

### DISCUSSION THREAD

**Discussion by strivedi-met** [Azure-Player/azure.datafactory.tools#197] [Q&A] [answered]
We are trying to deploy just the Triggers, without the dependencies (Pipeline etc) as part of a release branch. The extension is still looking for the referenced object, which are not part of this utility release branch. Is there an option to bypass this check ? Thank you

**Comment by NowinskiK** [comment, marked as answer]
First of all - build pipeline should wrap up all objects of a given ADF as artifact. So, even when you're making selective deployment (triggers only) - all other objects should be available there.
However, in some specific scenarios (perhaps like yours) there is an option that allows you to skip reference validation:
`IgnoreLackOfReferencedObject`
It's available among the other publish options in the PowerShell module (just noticed it must be documented)

Example:
```
$o = New-AdfPublishOption
$o.IgnoreLackOfReferencedObject = $false

Publish-AdfV2FromJson ...
```

**Reply by strivedi-met** [reply]
Thank you! How can be use the publish options with the azure.datafactory.devops tool?

**Reply by NowinskiK** [reply]
Unfortunately, the option is not yet available in the task for DevOps.
I will add it to the next version.
https://github.com/SQLPlayer/azure.datafactory.devops/issues/89

**Reply by strivedi-met** [reply]
Thank you.
